### PR TITLE
Hardcode gateway url

### PR DESCRIPTION
**Pull request by Xirado**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

This pull request hardcodes Discord's gateway URL.

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/api/utils/SessionController.java [hunk 1]
```diff
@@ -138,16 +138,14 @@ default void setConcurrency(int level) {}
     void setGlobalRatelimit(long ratelimit);
 
     /**
-     * Called by a JDA session when a new gateway session starts (Connecting, Reconnecting).
-     * <br>Should provide the gateway endpoint (wss) to connect to.
-     *
-     * @param  api
-     *         The current JDA instance (used for RestActions and ShardInfo)
-     *
+     * Discords gateway URL
```
Comment (DV8FromTheWorld) [inline src/main/java/net/dv8tion/jda/api/utils/SessionController.java:141]: These needs better docs.

Comment (Xirado) [reply src/main/java/net/dv8tion/jda/api/utils/SessionController.java:141]: will fix tomorrow

**Comment by JellyBrick** [comment]
Why hardcode gateway URL?

**Comment by Xirado** [comment]
> Why hardcode gateway URL?

because it is constant

**Comment by RedDaedalus** [comment]
image

**Review by MinnDevelopment — APPROVED** [review]

**Pull request merged**